layer0/build.js

```diff
@@ -10,6 +11,26 @@ module.exports = async function build(options) {
   try {
     builder.clearPreviousBuildOutput();
     let command = 'yarn build --target node';
+    let selectedAnswer = 'yarn';
+    if (!process.env.LAYER0_GITHUB_TOKEN) {
+      await inquirer
+        .prompt([
+          {
+            type: 'list',
+            name: 'Build your app with',
+            choices: ['npm', 'yarn'],
+          },
+        ])
+        .then((answers) => {
+          selectedAnswer = answers['Build your app with'];
+        })
+        .catch((e) => {
+          console.log(e);
+        });
+      if (selectedAnswer.includes('npm')) {
+        command = 'npm run build -- --target node';
+      }
+    }
```

Comment:
Instead of making the build process interactive, I would just check to see which package manager is being used.

You might be able to pull in the CLI utils for this:

```
const { isYarn } = require('@layer0/cli/utils/packageManager')
...
let command = 'npm run build -- --target node';
if (isYarn()) {
  command = 'yarn build --target node'
}
```

Or at the very least, check for `yarn.lock`.

Reply:
That's smort!
Thanks, Yep, updating that!